### PR TITLE
[TECH] Deploy HA Postgres Cluster

### PR DESCRIPTION
**Pull request by GuessWhoSamFoo**
XR #823 

Made edits to freelancer submission. Still pending author input.

### CODE REVIEW

**Comment by GuessWhoSamFoo** [comment]
[Deploying an HA PostgresSQL server with Backups - Bounty, K. Kabra](https://trello.com/c/lQFGCEG8/18-deploying-an-ha-postgressql-server-with-backups-bounty-k-kabra)

File: docs/databases/postgresql/how-to-deploy-ha-postgresql-cluster.md [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+
+2. Next, install Postgres:
+
+        sudo apt-get install postgresql-9.5 -y
+
+3. When you install Postgres, it also starts running as a service. We need to stop Postgres so that Patroni can manage it from this point on:
+
+        sudo systemctl stop postgresql
+
+4. Patroni manages utilities that comes default with Postgres. In Ubuntu 16.04, these utilities are installed in the `/usr/lib/postgresql/9.5/bin` directory by default.
```
Comment (kulshekhar) [inline docs/databases/postgresql/how-to-deploy-ha-postgresql-cluster.md]: This should be something to the effect of "Patroni uses some utilities that comes with Postgres, to manage Postgres"

**Comment by GuessWhoSamFoo** [comment]
[Deploy an HA Postgres server with Backups - Bounty, K. Kabra](https://trello.com/c/hIxt4LAb/442-deploy-an-ha-postgres-server-with-backups-bounty-k-kabra)

**Comment by kulshekhar** [comment]
Am I supposed to have access to the Trello board being linked to or should I ignore those?

**Comment by kcraig34** [comment]
@GuessWhoSamFoo, @EdwardAngert  -- Trying to track down the status of this guide.

**Comment by GuessWhoSamFoo** [comment]
@kcraig34 Merged copy edit. Ready for SEA edit/graphic

**Comment by kcraig34** [comment]
Thanks, @GuessWhoSamFoo --
@EdwardAngert -- I'll do an SEO edit on this, so as to not interfere with what you're doing at the moment. I'd like you to review it when I'm done.
Thanks, guys.

**Pull request merged**